bundles/org.openhab.binding.wlanthermo/src/main/java/org/openhab/binding/wlanthermo/internal/WlanThermoNanoHandler.java

```diff
@@ -0,0 +1,203 @@
+                if (state != null) {
+                    updateState(channel.getUID(), state);
+                } else {
+                    // if we could not obtain a state, try trigger instead
+                    String trigger = wlanThermoNanoCommandHandler.getTrigger(channel.getUID(), data);
+                    if (trigger != null) {
+                        triggerChannel(channel.getUID(), trigger);
+                    }
+                }
+            }
+        } catch (URISyntaxException | InterruptedException | ExecutionException | TimeoutException e) {
+            logger.debug("Update failed, checking connection", e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Update failed, reconnecting...");
+            if (pollingScheduler != null) {
+                pollingScheduler.cancel(false);
+            }
+            for (Channel channel : thing.getChannels()) {
+                updateState(channel.getUID(), UnDefType.UNDEF);
+            }
+            checkConnection();
+        }
+    }
+
+    private void push() {
+        for (org.openhab.binding.wlanthermo.internal.api.nano.data.Channel c : data.getChannel()) {
+            try {
+                String json = gson.toJson(c);
+                logger.debug("Pushing: {}", json);
+                URI uri = config.getUri("/setchannels");
+                int status = httpClient.POST(uri).content(new StringContentProvider(json), "application/json")
+                        .timeout(5, TimeUnit.SECONDS).send().getStatus();
+                if (status == 401) {
+                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                            "No or wrong login credentials provided. Please configure username/password for write access to WlanThermo!");
+                    logger.warn(
+                            "No or wrong login credentials provided. Please configure username/password for write access to WlanThermo!");
+                } else if (status != 200) {
+                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Failed to update channel "
+                            + c.getName() + " on device, Statuscode " + status + " on URI " + uri.toString());
+                    logger.warn("Failed to update channel {} on device, Statuscode {} on URI {}", c.getName(), status,
+                            uri.toString());
+                } else {
+                    updateStatus(ThingStatus.ONLINE);
+                }
+            } catch (InterruptedException | TimeoutException | ExecutionException | URISyntaxException e) {
+                updateStatus(ThingStatus.ONLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        "Failed to update channel " + c.getName() + " on device!");
+                logger.warn("Failed to update channel {} on device", c.getName(), e);
```

Comment:
See above, thing status changes are always logged. You could reduce the exception stack trace to DEBUG or TRACE